go.mod

```diff
@@ -4,10 +4,15 @@ go 1.15
 
 require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.2.0
+	github.com/go-logr/logr v0.2.1 // indirect
 	github.com/onsi/ginkgo v1.13.0 // indirect
-	k8s.io/api v0.19.0
+	golang.org/x/net v0.0.0-20200904194848-62affa334b73 // indirect
+	golang.org/x/tools v0.0.0-20200904185747-39188db58858 // indirect
 	k8s.io/apimachinery v0.19.0
 	k8s.io/client-go v0.19.0
+	k8s.io/gengo v0.0.0-20200728071708-7794989d0000 // indirect
+	k8s.io/klog/v2 v2.3.0 // indirect
+	k8s.io/kube-openapi v0.0.0-20200831175022-64514a1d5d59 // indirect
```

Comment:
Do we need all these additions just to be able to add metav1.Condition?

Reply:
These are indirect dependencies, so I think it's just that I ran `go mod tidy` in the PR but not in a previous one. Probably should add a verifier so it doesn't go stale.